CMakeLists.txt

```diff
@@ -162,7 +162,9 @@ set(BOOST_COMPONENTS filesystem system)
 add_definitions(-DBOOST_ALL_NO_LIB=1)
 find_package(Boost REQUIRED COMPONENTS ${BOOST_COMPONENTS})
 
-find_path(HALF_INCLUDE_DIR half.hpp)
+find_path(HALF_INCLUDE_DIR half.hpp HINTS ${CMAKE_INSTALL_PREFIX}/include ${CMAKE_INSTALL_PREFIX}/include/half )
+message(STATUS "HALF_INCLUDE_DIR: ${HALF_INCLUDE_DIR}")
+string(APPEND CMAKE_CXX_FLAGS " -I${HALF_INCLUDE_DIR}")
```

Comment:
@trixirt similar to my comment in MIOpen repo. Can we achieve same results without manually modifying `CMAKE_CXX_FLAGS`?